src/components/calcite-shell-panel/calcite-shell-panel.tsx

```diff
@@ -86,25 +142,222 @@ export class CalciteShellPanel {
   }
 
   render(): VNode {
-    const { collapsed, detached, position } = this;
+    const {
+      collapsed,
+      detached,
+      position,
+      initialContentWidth,
+      contentWidth,
+      contentWidthMax,
+      contentWidthMin,
+      intlResize,
+      resizable
+    } = this;
 
     const contentNode = (
-      <div class={{ [CSS.content]: true, [CSS.contentDetached]: detached }} hidden={collapsed}>
+      <div
+        class={{ [CSS.content]: true, [CSS.contentDetached]: detached }}
+        hidden={collapsed}
+        ref={this.storeContentEl}
+        style={resizable && contentWidth ? { width: `${contentWidth}px` } : null}
+      >
         {this.renderHeader()}
         <div class={CSS.contentBody}>
           <slot />
         </div>
       </div>
     );
 
+    const separatorNode = resizable ? (
+      <div
+        aria-label={intlResize}
+        aria-orientation="horizontal"
+        aria-valuemax={contentWidthMax}
+        aria-valuemin={contentWidthMin}
+        aria-valuenow={contentWidth ?? initialContentWidth}
+        class={CSS.separator}
+        onKeyDown={this.separatorKeyDown}
+        ref={this.connectSeparator}
+        role="separator"
+        tabIndex={0}
+        touch-action="none"
+      />
+    ) : null;
+
     const actionBarNode = <slot name={SLOTS.actionBar} />;
 
-    const mainNodes = [actionBarNode, contentNode];
+    const mainNodes = [actionBarNode, contentNode, separatorNode];
 
     if (position === "end") {
       mainNodes.reverse();
     }
 
     return <div class={{ [CSS.container]: true }}>{mainNodes}</div>;
   }
+
+  // --------------------------------------------------------------------------
+  //
+  //  private Methods
+  //
+  // --------------------------------------------------------------------------
+
+  setContentWidth(width: number): void {
+    const { contentWidthMax, contentWidthMin, position } = this;
+
+    const rounded = Math.round(position === "end" ? width : width);
```

Comment:
This is rounding the same value (`width`). Is there another value that should be used for rounding in the ternary?

Reply:
I think this is just a 🧠 💨 

:)